tools/run_tests/run_xds_tests.py

```diff
@@ -478,6 +482,21 @@ def wait_until_all_rpcs_go_to_given_backends(backends,
                                    allow_failures=False)
 
 
+def wait_until_no_rpcs_go_to_given_backends(backends, timeout_sec):
```

Comment:
Can this just use `wait_until_all_rpcs_go_to_given_backends_or_fail` instead of defining a new method?

Reply:
no don't see how to make that happen yet because the logic in the middle is completely opposite